recipes/textacy/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "textacy" %}
+{% set version = "0.3.3" %}
+{% set sha256sum = "8ab08d6cab17d05a3ee8a1d924ddbc2cdd57b668a180be43e3ba8f182b537e4e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  # At this time there is no spacy packages for win64-py27
+  skip: True  # [win32 or linux32 or (win and py2k)]
+
+# Numpy version supported >=1.8
+requirements:
+  build:
+    - python
+    - setuptools
+    - backports.csv >=1.0.1  # [py2k]
+    - cachetools >=2.0
+    - cld2-cffi >=0.1.4
+    - cytoolz >=0.8
+    - python-ftfy >=4.2
+    - fuzzywuzzy >=0.11
+    - ijson >=2.3
+    - matplotlib >=1.5
+    - networkx >=1.11
+    - numpy x.x
+    - pyemd >=0.3.0
+    - pyphen >=0.9.4
+    - python-levenshtein >=0.12.0
+    - requests >=2.10.0
+    - scipy >=0.17
+    - scikit-learn >=0.17
+    - spacy >=1.1
+    - unidecode >=0.04.19
```

Comment:
Please cu this list down to what is strictly needed to build. Guessing `python`, `setuptools`, and `numpy`.